selfhost/main.jakt

```diff
@@ -397,7 +397,13 @@ function main(args: [String]) {
         )
 
         if run_executable and compiler_status == 0 {
-            system(output_filename.c_string())
+            return system(output_filename.c_string())
+        } else {
+            if compiler_status == 0 {
+                return 0
+            } else {
+                return 1
+            }
```

Comment:
🤔 
```
        if ... {
            return system(...)
        }
        return compiler_status
```
?

Reply:
Doesn't work, unfortunately. `compiler_status` gets set to -1 because of `system`, but returning this directly on macOS turns it into a `0` 🤷 . So I use an additional step to normalise to 0 or 1.

Reply:
If we set this to a unique error status from jakt compiler error then the rationale for keeping run-one.sh in jakttest decreases 🤔

Reply:
Correction `system` was returning "status: 256"

Reply:
ah I see. okay then :shrug: